### ISSUE TITLE
Paperless-ngx paperless-scheduler fails to start

### ISSUE THREAD

**Issue by faheus** [tteck/Proxmox#785]
**Describe the issue**
paperless-scheduler.service fails with "Unknown command: 'qcluster'"
Fails even in a fresh LXC with no mods and default settings. 

`
Using Default Settings
Using Container Type: Unprivileged NO DEVICE PASSTHROUGH
Using Root Password: Automatic Login
Using Container ID: 114
Using Hostname: paperless-ngx
Using Disk Size: 4GB
Allocated Cores 2
Allocated Ram 2048
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway Address: Default
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Creating a Paperless-ngx LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 114 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.2.103 2003:c0:171d:d800:471:85ff:fe15:35bb
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.121.3
 ✓ Updated Container OS
 ✓ Installed Paperless-ngx Dependencies
 ✓ Installed OCR Dependencies
 ✓ Installed Extra Dependencies
 ✓ Installed JBIG2
 ✓ Downloaded Paperless-ngx
 ✓ Set up database
 ✓ Set up admin Paperless-ngx User & Password
 ✓ Finished installing Paperless-ngx
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

Paperless-ngx should be reachable by going to the following URL.
         http://192.168.2.103:8000

root@hive:~# pct enter 114
root@paperless-ngx:~# systemctl paperless-scheduler
Unknown command verb paperless-scheduler.
root@paperless-ngx:~# systemctl status paperless-scheduler
● paperless-scheduler.service - Paperless consumer
     Loaded: loaded (/etc/systemd/system/paperless-scheduler.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Mon 2022-11-28 17:07:48 UTC; 1min 2s ago
   Main PID: 19044 (code=exited, status=1/FAILURE)
        CPU: 1.018s

Nov 28 17:07:46 paperless-ngx systemd[1]: Started Paperless consumer.
Nov 28 17:07:48 paperless-ngx python3[19044]: Unknown command: 'qcluster'
Nov 28 17:07:48 paperless-ngx python3[19044]: Type 'manage.py help' for usage.
Nov 28 17:07:48 paperless-ngx systemd[1]: paperless-scheduler.service: Main process exited, code=exited, status=1/FAILURE
Nov 28 17:07:48 paperless-ngx systemd[1]: paperless-scheduler.service: Failed with result 'exit-code'.
Nov 28 17:07:48 paperless-ngx systemd[1]: paperless-scheduler.service: Consumed 1.018s CPU time.`

**Screenshots**

**Bug reports will be closed after 12 hours of NO activity.**

**Comment by tteck** [comment]
@Donkeykong307 would you take a look at this?

**Comment by tteck** [comment]
According to https://github.com/paperless-ngx/paperless-ngx/blob/main/scripts/paperless-scheduler.service
The `paperless-scheduler.service` was incorrect in the script. It has been corrected.

**Comment by faheus** [comment]
There is still something wonky, the consumer happily adds documents to the queue, but the queue gets never processed.

**Comment by euro2** [comment]
I have the same issue. All default, What ever i drag and drop, it gets uploaded and put in the queue, but the queue never gets processed. When i go find the file in the terminal i cannot find it anywhere.  Also when i put a .txt file in the /opt/paperless/consume dir and refresh, it is seen at put in the queue, but also not processed

EDIT: All 3 services are running

**Comment by tteck** [comment]
Something changed in the new version (1.10.0) Everything worked with version 1.9.2, even the `paperless-scheduler.service` was correct. You can test version 1.9.2 here
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/paperless-ngx-v4.sh)"
```

**Comment by tteck** [comment]
Switched the main branch back to using v1.9.2 until a solution is found.

**Comment by tteck** [comment]
Script is now fixed! e3ebc43ae6fd685745c52260ea82cb56119f575e

**Comment by tteck** [comment]
The update script should now successfully update to the latest version 165ea5897bc8246c62aa6baac9652cc2e556138a

**Comment by faheus** [comment]
> Script is now fixed! [e3ebc43](https://github.com/tteck/Proxmox/commit/e3ebc43ae6fd685745c52260ea82cb56119f575e)

Works good now! Thanks!

**Comment by euro2** [comment]
working!
